src/main/java/com/ctc/wstx/api/ReaderConfig.java

```diff
@@ -418,6 +422,11 @@ public final class ReaderConfig
      * references
      */
     protected URL mBaseURL;
+    
+    /**
+     * Resolve surrogate pairs of entities (2 code-points as one target character)
+     */
+    protected boolean mResolveEntitySurrogatePairs = Boolean.FALSE;
```

Comment:
`false` ?

Reply:
Due to task specification from the issue #165. Quotation:

> If you have time and interest, adding new configuration property that would allow inclusion of surrogates could be acceptable as well: **something disabled by default, but that can be enabled**.

Reply:
Ah, sure.  My point was that, given the property is the scalar type `boolean`, the default value could `false` instead of `Boolean.FALSE`.  No big deal though...

Reply:
ahhhh. Yes, that makes sense to me. It should be `false` indeed.